src/routes/about/+page.svelte

```diff
@@ -1 +1,260 @@
+			linkedin: 'https://www.linkedin.com/in/yonasworku/'
+		},
+		{
+			name: 'Matthew DeGuzman',
+			position: 'Technical Events Coordinator',
+			image: 'e-board/Matthew_DeGuzman.png',
+			linkedin: 'https://www.linkedin.com/in/matthew-deguzman/'
+		},
+		{
+			name: 'Alex Huper',
+			position: 'Academic Programs Coordinator',
+			image: 'e-board/Alex_Huper.png',
+			linkedin: 'https://www.linkedin.com/in/alex-huper/'
+		},
+		{
+			name: 'Josue Vicente',
+			position: 'Academic Events Coordinator',
+			image: 'e-board/Josue_Vicente.png',
+			linkedin: 'https://www.linkedin.com/in/josue-e-vicente/'
+		},
+		{
+			name: 'Abigail Erefah',
+			position: 'Digital Marketing Coordinator',
+			image: 'e-board/Abigail_Erefah.png',
+			linkedin: 'https://www.linkedin.com/in/abigail-erefah/'
+		},
+		{
+			name: 'Dr. Sanethia Thomas',
+			position: 'Faculty Advisor',
+			image: 'e-board/Sanethia_Thomas.png',
+			linkedin: 'https://www.linkedin.com/in/sanethia-thomas-phd-112806/'
+		}
+	];
+
+	let first_name = '';
+	let last_name = '';
+	let email = '';
+	let message = '';
+
+	async function submitForm() {
+		const response = await fetch('/about', {
+			method: 'POST',
+			headers: {
+				'Content-Type': 'application/json'
+			},
+			body: JSON.stringify({
+				first_name,
+				last_name,
+				email,
+				message
+			})
+		});
+
+		if (response.ok) {
+			alert('Email sent successfully');
+		} else {
+			alert('Failed to send email');
+		}
+	}
+</script>
+
+<div class="bg-body-background-white">
+	<!-- ColorStack UF intro -->
+	<div class="h-[40rem] h-auto laptop:h-auto lg-desktop:max-w-[1920px] mx-auto">
+		<div class="relative flex flex-col items-end laptop:items-start bg-[url('graphics/about-intro.png')] bg-cover bg-no-repeat bg-center h-[25rem] laptop:h-[45rem]">
+			<div class="relative grid grid-cols-1 top-[16rem] laptop:top-0 laptop:left-[3rem] laptop:top-[5rem] z-10 bg-body-background-blue pl-7 pr-4 py-7 laptop:p-12 rounded-l-lg laptop:rounded-lg mb-12 w-11/12 min-w-[310px] h-[380px] laptop:w-[460px] laptop:h-auto">
+				<h1 class="text-white text-lg tablet:text-2xl laptop:text-3xl font-gotham-medium laptop:pb-4 ">
+					ColorStack UF is a team committed to...
+				</h1>
+				<p class="text-white font-archer text-lg tablet:text-xl leading-6 self-start">
+					Investing in the futures of those left behind every day. We’re a tech student organization
+					cultivating the strongest campus community of its kind to deliver community building,
+					academic support, and career development opportunities.
+                </p>
+				<a href="https://linktr.ee/colorstackuf" target="_blank" class="self-end laptop:self-start">
+					<div class="text-black w-44 text-center bg-colorstackuf-orange transition-bg-color transition-color hover:bg-colorstackuf-blue hover:text-white duration-300 py-3 px-7 rounded-3xl mt-8 font-gotham-medium">
+						View Link Tree
+					</div>
+				</a>
+			</div>
+		</div>
+	</div>
+
+	<!-- About ColorStack National -->
```

Comment:
Make the font larger once you zoom out a bit more. It becomes unreadable on larger screensizes. Do this for the "ColorStack UF is a team committed to..." section as well as the positions for the e-board members.

Example:
<img width="1440" alt="image" src="https://github.com/user-attachments/assets/5a12be24-eac8-4695-9e03-43866d535f26">